serde/src/de/impls.rs

```diff
@@ -710,88 +693,91 @@ tuple_impls! {
 
 macro_rules! map_impl {
     (
-        $ty:ty,
-        $visitor_ty:ident < $($typaram:ident : $bound1:ident $(+ $bound2:ident)*),* >,
-        $visitor:ident,
-        $ctor:expr,
+        $ty:ident < $($typaram:ident : $bound1:ident $(+ $bound2:ident)*),* >,
         $with_capacity:expr
     ) => {
-        /// A visitor that produces a map.
-        pub struct $visitor_ty<$($typaram),*> {
```

Comment:
[This is a breaking change.](https://github.com/search?l=Rust&q=%22impls%3A%3ABTreeMapVisitor%22&ref=searchresults&type=Code&utf8=%E2%9C%93)